cdr/cdr_pgsql.c

```diff
@@ -783,12 +783,19 @@ static int config_module(int reload)
 
 static int load_module(void)
 {
-	ast_cli_register_multiple(cdr_pgsql_status_cli, sizeof(cdr_pgsql_status_cli) / sizeof(struct ast_cli_entry));
+    int res = 0;
+
 	if (config_module(0)) {
-		return AST_MODULE_LOAD_DECLINE;
+		res = AST_MODULE_LOAD_DECLINE;
+	} else {
+		if(ast_cdr_register(name, ast_module_info->description, pgsql_log)) {
+			res = AST_MODULE_LOAD_DECLINE;
+		} else {
+			ast_cli_register_multiple(cdr_pgsql_status_cli, sizeof(cdr_pgsql_status_cli) / sizeof(struct ast_cli_entry));
```

Comment:
You can replace the `sizeof()` calculation with just:

    ast_cli_register_multiple(cdr_pgsql_status_cli, ARRAY_LEN(cdr_pgsql_status_cli));

Reply:
Done, thanks!